dist/emoji-en-US.json

```
@@ -7606,7 +7635,8 @@
     "cheers",
     "clink",
     "drinks",
-    "mug"
+    "mug",
+    "write_drunkenly"
```

```suggestion
    "mug"
```

drunk is already a keyword.